_includes/1.0/api/creditmemo.html

```diff
@@ -55,12 +55,26 @@ <h3 id="creditmemojsonformat">JSON Format</h3>
                 <td>Remaining amount to be paid.</td>
             </tr>
             <tr>
-                <td>customer_id</td>
+                <td>customer_id <span class="deprecated">DEPRECATED</span></td>
                 <td>{{ site.format.type.integer }}</td>
                 <td>yes</td>
                 <td>no</td>
                 <td>Id of the <a href="/docs/1.0/api/#customer" title="">Customer.</a></td>
             </tr>
+            <tr>
+                <td>customer</td>
+                <td>{{ site.format.type.snippet_customer }}</td>
```

Comment:
Related to the previous comment, here we will have `site.format.type.snippet`, linked to the common "snippet" section of the documentation.